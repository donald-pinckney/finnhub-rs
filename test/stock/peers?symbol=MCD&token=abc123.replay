{"request":{"url":"https://finnhub.io/api/v1/stock/peers?symbol=MCD&token=abc123","method":"GET","body":null,"headers":{}},"response":{"url":"https://finnhub.io/api/v1/stock/peers?symbol=MCD&token=abc123","status":200,"headers":{"access-control-allow-credentials":"true","access-control-allow-headers":"Origin","access-control-allow-methods":"POST, GET, OPTIONS, PUT, DELETE","access-control-allow-origin":"*","access-control-expose-headers":"Content-Length","alt-svc":"h3-27=\":443\"; ma=86400, h3-28=\":443\"; ma=86400, h3-29=\":443\"; ma=86400","cf-cache-status":"DYNAMIC","cf-ray":"644c3cf72febc540-ORD","cf-request-id":"09a3846e7b0000c54098329000000001","connection":"keep-alive","content-type":"application/json; charset=utf-8","date":"Sat, 24 Apr 2021 03:27:50 GMT","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","nel":"{\"max_age\":604800,\"report_to\":\"cf-nel\"}","report-to":"{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report?s=EKGm0ta%2BnOGo2vmqjWZKAuVmSOurrpByPutlI13L9rquZ8XH4ruM0LM62OMZ2KPzS%2F5mnIuPH3mJnicd8MHqXakCCxQyKLt5LBN0%2FZddRqMeupICOyso\"}],\"max_age\":604800,\"group\":\"cf-nel\"}","server":"cloudflare","set-cookie":"__cfduid=db81dde45a8ce40448f552722aed6e9c31619234870; expires=Mon, 24-May-21 03:27:50 GMT; path=/; domain=.finnhub.io; HttpOnly; SameSite=Lax; Secure","transfer-encoding":"chunked","x-ratelimit-limit":"60","x-ratelimit-remaining":"53","x-ratelimit-reset":"1619234930"},"body":"WyJNQ0QiLCJTQlVYIiwiQ01HIiwiWVVNIiwiRFJJIiwiRFBaIiwiQVJNSyIsIlRYUkgiLCJXRU4iLCJTSEFLIl0="},"format_version":3}