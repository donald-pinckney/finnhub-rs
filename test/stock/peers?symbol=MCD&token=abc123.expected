[
    "MCD",
    "SBUX",
    "CMG",
    "YUM",
    "DRI",
    "DPZ",
    "ARMK",
    "TXRH",
    "WEN",
    "SHAK",
]
